packages/commonwealth/client/scripts/views/components/AddToHomeScreenPrompt/AddToHomeScreenPrompt.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useState } from 'react';
+import { CWCheckbox } from '../component_kit/cw_checkbox';
+import { CWIcon } from '../component_kit/cw_icons/cw_icon';
+import { CWText } from '../component_kit/cw_text';
+import { CWButton } from '../component_kit/new_designs/cw_button';
+import './AddToHomeScreenPrompt.scss';
+
+interface AddToHomeScreenPromptProps {
+  isIOS: boolean;
+  isAndroid: boolean;
+}
+
+export const AddToHomeScreenPrompt = ({
+  isIOS,
+  isAndroid,
+}: AddToHomeScreenPromptProps) => {
+  const [showPrompt, setShowPrompt] = useState(true);
+  const [checkboxChecked, setCheckboxChecked] = useState(false);
+  const HIDE_PROMPT_TIME = 'hidePromptTime';
+  const HIDE_PROMPT_DAYS = 'hidePromptDays';
+  const HIDE_PROMPT = 'hidePrompt';
+
+  useEffect(() => {
+    const hidePromptTime = localStorage.getItem(HIDE_PROMPT_TIME);
+    if (hidePromptTime && new Date().getTime() < Number(hidePromptTime)) {
+      setShowPrompt(false);
+    }
+
+    if (sessionStorage.getItem(HIDE_PROMPT)) {
+      setShowPrompt(false);
+    }
+  }, []);
+
+  const hidePromptForNDays = () => {
+    const maxDays = 30;
+
+    let n = Number(localStorage.getItem(HIDE_PROMPT_DAYS)) || 1;
+    n = n * 2 > maxDays ? maxDays : n * 2;
+    const hideUntil = new Date().getTime() + n * 24 * 60 * 60 * 1000;
+    localStorage.setItem(HIDE_PROMPT_TIME, hideUntil.toString());
+    localStorage.setItem(HIDE_PROMPT_DAYS, n.toString());
+    setShowPrompt(false);
+  };
+
+  const iosPrompt = () => {
+    return (
+      <div className="ios-home-screen-prompt">
+        <div className="prompt-content">
+          <div className="header">
+            <div className="icon">
+              <img src="/static/img/branding/common.svg" alt="Commonwealth" />
+            </div>
+            <CWText className="title">Add to Home Screen</CWText>
+          </div>
+          <CWText className="description">
+            For the best mobile experience we recommend installing the Common
+            web-app.
+          </CWText>
+          <div className="instructions">
+            <CWText className="instruction">
+              1. Tap the share{' '}
+              <CWIcon className="share-icon" iconName="export" /> icon below
+            </CWText>
+            <CWText className="instruction">
+              2. Select <span className="highlight">Add to Home Screen</span>
+            </CWText>
+          </div>
+          <CWButton
+            buttonType="tertiary"
+            label="Show less often"
+            containerClassName="hide-prompt"
+            onClick={() => hidePromptForNDays()}
+          />
+        </div>
+      </div>
+    );
+  };
+
+  const androidPrompt = () => {
```

Comment:
in previous PR it was a bit less complex but in this case I would like to split this into smaller components. So here is what I would ask you to change:

In `AddToHomeScreenPrompt` directory add:
1. `AndroidPrompt` directory
  - AndroidPrompt.tsx => top parent css class will be `AndroidPrompt`
  - AndroidPrompt.scss
  - index.ts
2. `IOSPrompt` directory
  - IOSPrompt.tsx => top parent css class will be `IOSPrompt`
  - IOSPrompt.scss
  - index.ts

use those components in `AddToHomeScreenPrompt`

Reply:
Got it, will split it into different files.